6. Objects and Methods /instance_variable_example.rb

```diff
@@ -6,17 +6,16 @@ def initialize(id, name, addr)
       @cust_name = name
       @cust_addr = addr
    end
-   def display_details()
-      puts "Customer id #@cust_id"
-      puts "Customer name #@cust_name"
-      puts "Customer address #@cust_addr"
-   end
-end
+
+   def to_s
+    "(#{@cust_id}, #{@cust_id}, #{@cust_id})"
```

Comment:
As per our string interpolation logic, without the @,  why is there is an error `to_s': undefined local variable or method `cust_id' for #<Customer:0x00007ffb1e9e2b88> (NameError)

Reply:
because there is no attr reader for `cust_id`